### PR TITLE
[All Hosts] Add articles about developing and publishing add-ins using VS Code.

### PR DESCRIPTION
**Pull request by kbrandl**
This PR adds a new article about developing Office Add-ins with VS Code (in the **Develop** section of the TOC) and also a new article about publishing an Office Add-in that was developed with VS Code (in the **Publish** section of the TOC). It also slightly refactors the corresponding Visual Studio article (for consistency in naming).

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2cdfa68](https://github.com/OfficeDev/office-js-docs-pr/commits/2cdfa68f5fdee51fe544eea86a8f6537cfe37fce)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/publish/host-an-office-add-in-on-microsoft-azure.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/host-an-office-add-in-on-microsoft-azure.md) | :bulb:Suggestion | | [Details](#user-content-47fe08dd-1811fd97)
[docs/develop/develop-add-ins-vscode.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/develop/develop-add-ins-vscode.md) | :white_check_mark:Succeeded | |
[docs/includes/publish-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/includes/publish-add-in.md) | :white_check_mark:Succeeded | |
[docs/publish/package-your-add-in-using-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/package-your-add-in-using-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/publish/publish-add-in-vs-code.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/publish-add-in-vs-code.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="47fe08dd-1811fd97"></a>
### [docs/publish/host-an-office-add-in-on-microsoft-azure.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/host-an-office-add-in-on-microsoft-azure.md)
 - **Line 31**: **[Suggestion]** `````Link 'https://azure.microsoft.com/en-us/pricing/member-offers/visual-studio-subscriptions/' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites. NOTE: This Suggestion will become a Warning on 12/20/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C11%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201911272028278113-1457%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AF2NvLvg%2BXH4QXBBTesfi8t69iV%2B%2FAAj1b3OCZh0jdI%3D&st=2019-11-27T20%3A24%3A48Z&se=2019-12-28T20%3A29%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[dbeb299](https://github.com/OfficeDev/office-js-docs-pr/commits/dbeb2998990bc8f874c63e99bc2eb6f084fb93c8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/add-in-design.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/design/add-in-design.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-add-ins-vscode.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/develop/develop-add-ins-vscode.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/includes/alert-script-lab-support.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/includes/alert-script-lab-support.md) | :white_check_mark:Succeeded | |
[docs/includes/publish-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/includes/publish-add-in.md) | :white_check_mark:Succeeded | |
[docs/publish/host-an-office-add-in-on-microsoft-azure.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/host-an-office-add-in-on-microsoft-azure.md) | :white_check_mark:Succeeded | |
[docs/publish/package-your-add-in-using-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/package-your-add-in-using-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/publish/publish-add-in-vs-code.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/publish-add-in-vs-code.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912021737330795-1457%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TTBezEcamsq8%2FC%2FcNTXdtujGONcdQwUokX%2B%2B8qp3fgg%3D&st=2019-12-02T17%3A33%3A58Z&se=2020-01-02T17%3A38%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT just FYI, I'd like your feedback on this PR, whenever you have time. (I submitted it just before the holiday, so re-pinging now to ensure it's on your radar.) Thank you!

File: docs/publish/publish-add-in-vs-code.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Publish an add-in developed with Visual Studio Code
+description: Publish an add-in developed with Visual Studio Code
```
Comment (ElizabethSamuel-MSFT) [inline docs/publish/publish-add-in-vs-code.md]:
```suggestion
description: How to publish an add-in developed with Visual Studio Code.
```

File: docs/publish/publish-add-in-vs-code.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+ms.date: 11/27/2019
+localization_priority: Priority
+---
+
+# Publish an add-in developed with Visual Studio Code
+
+This article describes how to publish an Office Add-in that you created using the Yeoman generator and developed with [Visual Studio Code (VS Code)](https://code.visualstudio.com) or any other editor.
```
Comment (ElizabethSamuel-MSFT) [inline docs/publish/publish-add-in-vs-code.md]:
```suggestion
You can publish an Office Add-in that you created using the Yeoman generator and developed with [Visual Studio Code (VS Code)](https://code.visualstudio.com) or any other editor.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5728fb6](https://github.com/OfficeDev/office-js-docs-pr/commits/5728fb646a66b50fc58799bd1500f20b2c233da4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/design/add-in-design.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/design/add-in-design.md) | :white_check_mark:Succeeded | |
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/develop/develop-add-ins-vscode.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/develop/develop-add-ins-vscode.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-errors.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/excel/custom-functions-errors.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-troubleshooting.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/excel/custom-functions-troubleshooting.md) | :white_check_mark:Succeeded | |
[docs/includes/alert-script-lab-support.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/includes/alert-script-lab-support.md) | :white_check_mark:Succeeded | |
[docs/includes/publish-add-in.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/includes/publish-add-in.md) | :white_check_mark:Succeeded | |
[docs/publish/host-an-office-add-in-on-microsoft-azure.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/host-an-office-add-in-on-microsoft-azure.md) | :white_check_mark:Succeeded | |
[docs/publish/package-your-add-in-using-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/package-your-add-in-using-visual-studio.md) | :white_check_mark:Succeeded | |
[docs/publish/publish-add-in-vs-code.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/publish/publish-add-in-vs-code.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/kb-develop-with-vs-code/docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C2%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912021825484135-1457%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AWY6Sgr%2BoK50%2Brjli3mtSlLPwrFofXhJJDzcYvyWJE8%3D&st=2019-12-02T18%3A22%3A09Z&se=2020-01-02T18%3A27%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by kbrandl** [comment]
@ElizabethSamuel-MSFT thanks for the feedback!

**Pull request merged**